python/src/lib.rs

```diff
@@ -63,8 +70,36 @@ impl RawDeltaTable {
             .map_err(PyDeltaTableError::from_raw)?)
     }
 
-    pub fn files(&self) -> PyResult<Vec<String>> {
-        Ok(self._table.get_files().to_vec())
+    pub fn files(
```

Comment:
i recommend creating a separate method for this like you did with the rust struct for better readability and to avoid runtime option matching.

Reply:
👍